Clients/Xamarin.Interactive.Client.Mac.SimChecker/Xamarin.Interactive.Client.Mac.SimChecker.csproj

```diff
@@ -1,52 +1,31 @@
-<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{D495E2AE-BF3E-444B-B3D6-91EF14293831}</ProjectGuid>
+    <TargetFramework>net461</TargetFramework>
     <OutputType>Exe</OutputType>
-    <RootNamespace>Xamarin.Interactive.Mac.SimChecker</RootNamespace>
-    <AssemblyName>Xamarin.Interactive.Mac.SimChecker</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <ProductVersion>8.0.30703</ProductVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <PlatformTarget>x64</PlatformTarget>
   </PropertyGroup>
+
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug</OutputPath>
     <DefineConstants>DEBUG;CATALOG_API_ONLY</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <ExternalConsole>true</ExternalConsole>
-    <PlatformTarget>x64</PlatformTarget>
+    <OutputPath>bin\Debug</OutputPath>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release</OutputPath>
     <DefineConstants>CATALOG_API_ONLY</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <ExternalConsole>true</ExternalConsole>
-    <PlatformTarget>x64</PlatformTarget>
+    <OutputPath>bin\Release</OutputPath>
   </PropertyGroup>
+
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Xml.Serialization" />
     <Reference Include="System.Xml" />
   </ItemGroup>
+
   <ItemGroup>
-    <Compile Include="Program.cs" />
-    <Compile Include="..\..\Agents\Xamarin.Interactive\BuildInfo.cs">
-      <Link>BuildInfo.cs</Link>
-    </Compile>
-    <Compile Include="..\..\Agents\Xamarin.Interactive\I18N\Catalog.cs">
-      <Link>Catalog.cs</Link>
-    </Compile>
-    <Compile Include="..\..\Agents\Xamarin.Interactive\Versioning\ReleaseVersion.cs">
-      <Link>ReleaseVersion.cs</Link>
-    </Compile>
+    <Compile Include="..\..\Agents\Xamarin.Interactive\BuildInfo.cs" />
+    <Compile Include="..\..\Agents\Xamarin.Interactive\I18N\Catalog.cs" />
+    <Compile Include="..\..\Agents\Xamarin.Interactive\Versioning\ReleaseVersion.cs" />
+    <Compile Include="..\..\Agents\Xamarin.Interactive\Logging\**\*.cs" />
   </ItemGroup>
   <Import Project="..\Xamarin.Interactive.MTouch\Xamarin.Interactive.MTouch.projitems" Label="Shared" Condition="Exists('..\Xamarin.Interactive.MTouch\Xamarin.Interactive.MTouch.projitems')" />
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />

```

Comment:
I'm pretty sure you can reduce the entire project to

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net461</TargetFramework>
    <DefineConstants>CATALOG_API_ONLY</DefineConstants>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="..\..\Agents\Xamarin.Interactive\BuildInfo.cs" />
    <Compile Include="..\..\Agents\Xamarin.Interactive\I18N\Catalog.cs" />
    <Compile Include="..\..\Agents\Xamarin.Interactive\Versioning\ReleaseVersion.cs" />
    <Compile Include="..\..\Agents\Xamarin.Interactive\Logging\**\*.cs" />
  </ItemGroup>
  <Import Project="..\Xamarin.Interactive.MTouch\Xamarin.Interactive.MTouch.projitems" Label="Shared" Condition="Exists('..\Xamarin.Interactive.MTouch\Xamarin.Interactive.MTouch.projitems')" />
</Project>
```

However, I wonder if we should just link `Xamarin.Interactive.dll` at this point since it's not a weird PCL. I also wonder if we can just move the `Xamarin.Interactive.MTouch` into `Xamarin.Interactive.dll` entirely.

Reply:
Yeah, this looks better.

We could link it, or we could do multiple SSH copies, or use the existing mechanisms for transferring multiple files via zip+ssh... And I definitely want to get rid of the shared project in master.

Not sure if it's worth putting more time into this right now, though?